examples/mobile/UIComponents/components/content/index.html

```diff
@@ -23,6 +23,46 @@ <h1>
 						Contents area
 					</a>
 				</li>
+				<li class="ui-li-anchor">
+					<a href="floatingactions/index.html">
+						Floating Actions
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="textinput.html">
+						Text Field
```

Comment:
Text field has own category in new guideline.

Reply:
Yes. I know.
Now some Text field examples are not working well, so now I don't make their category.
I'll update the category in the next PR.
Thanks for your comments.